types/twine-sugarcube/macro.d.ts

```diff
@@ -119,8 +142,20 @@ export interface MacroContext {
 
 export interface MacroDefinition {
     handler: (this: MacroContext) => void;
+    /**
+     * Having this property signifies that this is a container macro
+     * This should be an array of child tag names or `null`
+     * @since 2.0.0
+     */
     tags?: string[] | null | undefined;
-    skipArgs?: boolean | undefined;
+    /**
+     * Disables parsing argument strings into discrete arguments.
+     * This is used by macros that only use the raw/full argument strings.
+     * `true` to affect all tags
+     * Or Array of tags to affect
+     * @since 2.0.0
+     */
+    skipArgs?: boolean | undefined | string[];
```

Comment:
Same as with `tags?` above.  I don't think we need an explicit `undefined` on an optional type.